src/components/dashboard/dashboardv1.js

```diff
@@ -0,0 +1,126 @@
+      .post(`/${history.location.state.userid}/uploading`, data, {
+        headers: {
+          "Content-Type": "multipart/form-data"
+        }
+      })
+      .then(res => {
+        console.log(res);
+        setRatings(res.data);
+      })
+      .catch(err => {
+        console.log(err);
+      });
+    // clears out previous uploaded file
+    data = new FormData();
+  };
+
+  useEffect(() => {
+    axios
+      .get(`https://api.groa.us/api/users/1111/recommendations`)
+      .then(res => setRecommendations(res.data.recommendation_json))
+      .catch(err =>
+        console.log("Something went wrong in fetching recommendations.", err)
+      );
+  }, []);
+
+  switch (true) {
+    case !history?.location?.state?.userid:
+      return <LoadingScreen />;
+    case !ratings && !recommendations?.length:
+      return (
+        <div className="bigContainer" data-test="dashboard-screen">
+          <div className="form-hover">
+            {/* will create a component in the future*/}
+            <form id="zip-form">
+              <input
+                id="zip-input"
+                className="movie-input"
+                type="file"
+                placeholder="letterbox csv file here"
+                name="movies"
+                value={input.movieName}
+                onChange={changeHandler}
+              />
+            </form>
+            <p className="form-directions">
+              upload your letterboxd csv file here to get all past movie ratings
+            </p>
+          </div>
+        </div>
+      );
+    default:
+      return (
+        <div className="bigContainer" data-test="dashboard-screen">
+          <div className="form-hover">
+            <form id="zip-form">
+              <input
+                id="zip-input"
+                className="movie-input"
+                type="file"
+                placeholder="letterbox csv file here"
+                name="movies" // name of file / name of obj?  used in key field for postman to upload a file too
+                value={input.movieName} // im not sure why this works but it does.  // document this portion out as well
+                onChange={changeHandler}
+              />
+            </form>
+            {Object.keys(ratings).length === 0 ? (
+              <p className="form-directions">
+                upload your letterboxd csv file here to get all past movie
+                ratings
+              </p>
+            ) : (
+              <p className="upload-successful">
+                <strong>{ratings.message}</strong>
+              </p>
+            )}
+          </div>
+
+          {/* this shows the recommendations movie cards */}
+          <h2>Your Recommendations</h2>
+          <div data-test="box-container" className="box-container">
+            {/* should off set up to 200 and then retun nothing */}
+            {recommendations.map((x, index) =>
+              index < 201 ? (
+                <MovieCard
```

Comment:
Created a movie card component instead of updating each of the divs individually.